src/index.html

```diff
@@ -1,13 +1,546 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class = "page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>THE MET</title>
+    <link rel="icon" type="image/x-icon" href="../src/images/met.svg">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&family=Roboto&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page__body">
+    <header class="header" id="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <a href="#header" class="top-bar__logo-link">
+              <img 
+                src="../src/images/met.svg" 
+                alt="Met logo" 
+                class="top-bar__logo"
+              >
+            </a>
+            <div class="top-bar__icons">
+              <a href="tel: +1 212-535-7710" class="icon icon--call"></a>
```

Comment:
Need to fix format phone only in href everywhere
```suggestion
              <a href="tel:+12125357710" class="icon icon--call"></a>
```